lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -171,8 +171,17 @@ def unstore(customer_vault_id, options = {})
       end
       alias_method :delete, :unstore
 
+      def network_tokenization_supported
```

Comment:
I understand the discussion that was previously held https://github.com/activemerchant/active_merchant/pull/2241#issuecomment-262103147, but I don't like the lack of query / command segregation here. 

Also, and I lack understanding around this, but doesn't Braintree Vault also use network tokenization? What if someone using the gem wants to ensure Braintree Blue supports network tokenization but isn't setup to provision apple pay? As an integrator, I'd call `supports_network_tokenization?` to ensure I can use Braintree Vault, but I'd get back false. Furthermore, I'd probably get an exception specific to Apple Pay provisioning, which is really confusing.